cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -51,7 +51,7 @@
 #  - Document functions using proper syntax:
 #    https://technet.microsoft.com/en-us/library/hh847834(v=wps.620).aspx
 
-$INFRA_CONTAINER = 'mcr.microsoft.com/k8s/core/pause:1.0.0'
+$INFRA_CONTAINER = 'gcr.io/gke-release/pause-win:1.0.0'
```

Comment:
To clarify: is this GCR bucket available to any user, not just google-owned projects?

Reply:
Yes, it is available to public.